nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/resources/docs/org.apache.nifi.atlas.reporting.ReportLineageToAtlas/additionalDetails.html

```diff
@@ -121,40 +121,41 @@ <h3 id="nifi-atlas-types">NiFi Atlas Types</h3>
+        <h3 id="namespaces">Namespaces</h3>
 
         <p>An entity in Atlas can be identified by its GUID for any existing objects, or type name and unique attribute can be used if GUID is not known. Qualified name is commonly used as the unique attribute.</p>
-        <p>Since one Atlas instance can be used to manage multiple environments, i.e clusters, Atlas has to manage objects in different clusters those may have the same name. For example, a Hive table 'request_logs' in a 'cluster-A' and 'cluster-B'. In such case, cluster name embedded in qualified names are crucial.</p>
+        <p>One Atlas instance can be used to manage multiple environments and objects in different environments may have the same name. For example, a Hive table 'request_logs' in two different clusters, 'cluster-A' and 'cluster-B'. For this reason the qualified names contain a so-called metadata namespace.</p>
+        <p>It's common practice to provide the cluster name as the namespace, but it can be any arbitrary string.</p>
 
-        <p>For these requirements, a qualified name has 'componentId@clusterName' format. E.g. A Hive table qualified name would be dbName.tableName@clusterName (default.request_logs@cluster-A).</p>
+        <p>With this, a qualified name has 'componentId@namespace' format. E.g. A Hive table qualified name would be dbName.tableName@namespace (default.request_logs@cluster-A).</p>
 
-        <p>From this NiFi reporting task standpoint, a cluster name is need to be resolved at following situations:
+        <p>From this NiFi reporting task standpoint, a namespace is needed to be resolved at following situations:
             <ul>
-                <li>To register NiFi component entities. Which cluster name should be used to represent the current NiFi cluster?</li>
-                <li>To create lineages from NiFi component to other DataSets. Which cluster does the DataSet resides?</li>
+                <li>To register NiFi component entities. Which namespace should be used to represent the current NiFi environment?</li>
+                <li>To create lineages from NiFi component to other DataSets. Which environment does the DataSet resides in?</li>
             </ul>
         </p>
 
-        <p>To answer such questions, ReportLineageToAtlas reporting task provides a way to define mappings from ip address or hostname to a cluster name.
-        The mapping can be defined by Dynamic Properties with a name in 'hostnamePattern.ClusterName' format, having its value as a set of Regular Expression Patterns to match ip addresses or host names to a particular cluster name.</p>
+        <p>To answer such questions, ReportLineageToAtlas reporting task provides a way to define mappings from ip address or hostname to a namespace.
+        The mapping can be defined by Dynamic Properties with a name in 'hostnamePattern.namespace' format, having its value as a set of Regular Expression Patterns to match ip addresses or host names to a particular cluster name.</p>
```

Comment:
"to a particular namespace." ?

Could you please also fix this old typo: ip address => IP address (several times around this section)